game ver.commit 1.js

```diff
@@ -225,7 +225,32 @@ function move() {
         dy = -dy;
     } else if (y + dy > canvas.height - ballRadius) {
         if (x > paddleX && x < paddleX + paddleWidth) { //공이 바닥이지만, 바에 맞을 경우
+            if(x > paddleX && x < (paddleX + paddleWidth)/7){
+                dx = -dx;
+                dx = dx * 1.15;
+            }
+            if (x > (paddleX + paddleWidth)/7 && x < (paddleX + paddleWidth)*2/7){
+                dx = -dx;
+                dx = dx * 1.1;
+            }
+            if (x > (paddleX + paddleWidth)*2/7 && x < (paddleX + paddleWidth)*3/7){
+                dx = -dx;
+                dx = dx * 1.05;
+            }
+            if (x > (paddleX + paddleWidth)*3/7 && x < (paddleX + paddleWidth)*4/7){
+                dx = dx*0.95;
+            }

```

Comment:
만약 위 코멘트가 괜찮다면 이건 어떤가요? if문 쭉 나열된 부분을 아래와 같이 바꿀 수 있어보여요

```suggestion
            for (let i = 0; i < 7; i++) {
                if (x > (paddleX + paddleWidth) * i / 7 && x < (paddleX + paddleWidth) * (i + 1) / 7) {
                    dx = -dx;
                    dx = dx + Math.abs((0.05 * (3 - i)));   // 1.15, 1.10, 1.05, 1.00, 1.05, 1.10, 1.15배 x증가폭 변경
                    break;
                }
            }

```